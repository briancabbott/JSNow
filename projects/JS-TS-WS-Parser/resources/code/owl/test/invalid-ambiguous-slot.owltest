---a = 
  b@identifier : b
 .operators prefix
  identifier : c
b = "b"