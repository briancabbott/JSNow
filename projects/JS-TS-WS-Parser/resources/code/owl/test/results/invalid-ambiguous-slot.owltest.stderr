error: in the rule 'a', the name 'identifier' could refer to two different rules

    b@identifier : b
    ~~~~~~~~~~~~    
  ...
    identifier : c
    ~~~~~~~~~~    

